NuGetPack.UnitTests/NuGetPackTests.cs

```diff
@@ -178,6 +178,23 @@ public void TestGetDependenciesInner()
                     && ((XElement)obj).Attribute("version").Value == "1.24.0");
         }
 
+        [TestMethod]
+        public void TestGetDependencies_InternalPackages()

```

Comment:
This test passes with and without the fix

Reply:
Oops, of course, it was a wrong test. Correction made.